cernopendata/modules/fixtures/data/docs/cms-guide-luminosity-calculation/cms-guide-luminosity-calculation.md

```diff
@@ -0,0 +1,292 @@
+# How to calculate luminosity for CMS open data
```

Comment:
I'll remove this line during merge as it would be having a "double h1 title" effect alongside the main page title that is there already.